addons/core/functions/fnc_initCBASettings.sqf

```diff
@@ -492,3 +492,11 @@
     "_independent",
     1
 ] call CBA_Settings_fnc_init;
+[
+    "TFAR_instantiate_asap", 
```

Comment:
I would prefer `instantiateAtBriefing` or with underscores. asap isn't very descriptive.

Reply:
I was just out of ideas. 😃